src/api/routes/subscriptions.js

```diff
@@ -597,17 +599,21 @@ subscriptions.deleteSubscription = function (app, res, applications, loggedInUse
                     }
                     res.status(204).send('');
 
-                    webhooks.logEvent(app, {
-                        action: webhooks.ACTION_DELETE,
-                        entity: webhooks.ENTITY_SUBSCRIPTION,
-                        data: {
-                            subscriptionId: subscriptionId,
-                            applicationId: appId,
-                            apiId: apiId,
-                            userId: loggedInUserId,
-                            auth: subscriptionData.auth
-                        }
-                    });
+                    const actionType = subsCreateInfo.ModifyPlan ? webhooks.ACTION_MODIFY : webhooks.ACTION_DELETE;
```

Comment:
change the ModifyPlan to camelcase pattern

Reply:
Okay